### PR TITLE
mkdir, touch, nmap: add Spanish translations

### PR DESCRIPTION
**Pull request by goznalo-git**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

Hi there, I've translated a couple of commands directly from their English counterparts. Some bits were somewhat awkward to translate, hence some technical terms were left intact (e.g. hostname). Please let me know how to proceed with them should I have to change those terms.

### CODE REVIEW

**Review by schneiderl — APPROVED** [review]
don't know the language but looks good to me

File: pages.es/common/nmap.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- Comprueba si una dirección IP está activa, e intenta averiguar el sistema operativo del servidor correspondiente:
+
+`nmap -O {{ip_o_hostname}}`
+
+- Intenta determinar si los hosts están activos y cuáles son sus nombres:
+
+`nmap -sn {{ip_o_hostname}} {{opcional_otra_direccion}}`
+
+- Como arriba, pero también ejecuta un escaneo de (por defecto) 1000 puertos TCP si el host está activo:
```
Comment (navarroaxel) [inline pages.es/common/nmap.md:15]:
```suggestion
- Como el anterior, pero también ejecuta un escaneo de 1000 puertos TCP por defecto, si el host está activo:
```

File: pages.es/common/nmap.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Escanea una lista específica de puertos (para todos los puertos 1-65535 usar -p-):
+
+`nmap -p {{puerto1,puerto2,…,puertoN}} {{direccion_o_direcciones}}`
+
+- Realiza un escaneo TCP y UDP (usar -sU para solo UDP, -sZ para SCTP, -sO para IP):
+
+`nmap -sSU {{direccion_o_direcciones}}`
+
+- Realiza un escaneo total de puertos, servicios, detección de versiones con todos los scrits NSE por defecto contra un host para determinar debilidades e información:
```
Comment (navarroaxel) [inline pages.es/common/nmap.md:35]:
```suggestion
- Realiza un escaneo total de puertos, servicios, detección de versiones con todos los scripts NSE por defecto contra un host para determinar debilidades e información:
```

File: pages.es/common/touch.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# touch
+
+> Cambia el tiempo de accesso y modificación de un archivo (atime, mtime).
```
Comment (navarroaxel) [inline pages.es/common/touch.md:3]:
```suggestion
> Cambia el tiempo de accesso y modificación de un archivo (atime, mtime).
> Más información: <https://man7.org/linux/man-pages/man1/touch.1.html>.
```

File: pages.es/common/mkdir.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Crea un directorio.
```
Comment (navarroaxel) [inline pages.es/common/mkdir.md:3]:
```suggestion
> Crea un directorio.
> Más información: <https://man7.org/linux/man-pages/man1/mkdir.1.html>.
```

File: pages.es/common/touch.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`touch {{archivo}}`
+
+- Establece los tiempos de un archivo a un dia y hora específicos:
+
+`touch -t {{YYYYMMDDHHMM.SS}} {{archivo}}`
+
+- Usa los tiempos de un archivo para establecer los tiempos en otro archivo:
+
+`touch -r {{archivo}} {{archivo2}}`
```
Comment (navarroaxel) [inline pages.es/common/touch.md:15]: We use long arguments when is possible:
```suggestion
`touch --reference={{archivo}} {{archivo2}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @goznalo-git, just a few comments. Could you please add the more information links to the english page too? (in this PR)

File: pages.es/common/touch.md (hunk 5 above)
Comment (bl-ue) [reply pages.es/common/touch.md:15]: In this case, it's **_not_** possible, so let's not do this. The original pages and all translations of it use the short argument, and one of the reasons for that is the ancient BSD `touch` bundled with macOS doesn't support it.
```console
$ touch file
$ touch --reference=file file2
touch: illegal option -- -
usage:
touch [-A [-][[hh]mm]SS] [-acfhm] [-r file] [-t [[CC]YY]MMDDhhmm[.SS]] file ...
$ sw_vers
ProductName:	macOS
ProductVersion:	11.2.2
BuildVersion:	20D80
$
```

**Review by goznalo-git — COMMENTED** [review]
Corrections accepted

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**